community_supplied/Kernel/kernel-tcpip/check-arp-cache-iri-count-iagent.rule

```diff
@@ -0,0 +1,181 @@
+/*
+ * This rule checks for the neighbour ARP cache count on IRI(Internal routing interface).
+ *
+ * It shows yellow alarm when the count exceeds 90 percent of the maximum count.
+ * It shows red alarm when the count exceeds 95 percent of the maximum count or if there is an increase in drop count.
+ * It's sampling time is 1 minute.
+ */
+iceberg {
+    topic kernel.tcpip {
+        rule check-arp-cache-iri-count-iagent {
+            synopsis "Check arp iri cache count";
+            description "Check arp iri cache count";
```

Comment:
Please make description more descriptive. e.g. This rule collects arp iri cache count and notify anomaly if arp iri cache count is not increasing.